roles/openshift_logging_fluentd/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 __latest_fluentd_version: "3_5"
 __allowed_fluentd_versions: ["3_5", "3_6"]
 __allowed_fluentd_types: ["hosted", "secure-aggregator", "secure-host"]
+openshift_logging_fluentd_buffer_type: "file"
```

Comment:
like below, will need to make sure we load `openshift_logging_fluentd_buffer_type` into a fact afterwards